CHANGELOG/CHANGELOG-1.12.md

```diff
@@ -1,5 +1,16 @@
 # Changelog 1.12
 
+## 1.12.1 - 2023-06-27
+
+### Fixed
+
+- Bump up libOpenflow and ofnet versions to fix a PackintIn2 response parse error. ([#5154](https://github.com/antrea-io/antrea/pull/5154), [@wenyingd])
+- Fix incorrect FlowMod message passing in openflow client modifyFlows to avoid unexpected flow error. ([#5125](https://github.com/antrea-io/antrea/pull/5125), [@Dyanngg])
+- Fix Egress IP not advertised issue when userspace ARP responder is not running or an Egress IP is obtained by multiple Nodes in some situations. ([#5127](https://github.com/antrea-io/antrea/pull/5127), [@tnqn])
```

Comment:
Ensure the Egress IP is always correctly advertised to the network, including when the userspace ARP responder is not running or when the Egress IP is temporarily claimed by multiple Nodes.

Reply:
Done